plugin.py

```diff
@@ -7,8 +7,8 @@
 from LSP.plugin.core.protocol import Location
 
 
-server_name = 'metals-sublime'
-settings_file = 'metals-sublime.sublime-settings'
+server_name = 'LSP-metals'
+settings_file = 'LSP-metals.sublime-settings'
```

Comment:
don't forget to rename the actual `metals-sublime.sublime-settings` to `LSP-metals.sublime-setting`